Accessibility. Crash on text hover on Windows

Compose 1.5.0, OpenJDK 19

On 1.4.3 it works well

OpenJDK 19

1. Run this example:
```
import androidx.compose.material.Text
import androidx.compose.ui.window.singleWindowApplication

fun main() = singleWindowApplication {
    Text("Text")
}
```
2. [Enable Java Access bridge](https://github.com/JetBrains/compose-multiplatform/tree/master/tutorials/Accessibility#windows), run NVDA 2023.2

3. Hover the text.

See the crash:
```
Exception in thread "Thread-1" java.lang.RuntimeException: java.lang.NullPointerException
	at jdk.accessibility/com.sun.java.accessibility.internal.AccessBridge$InvocationUtils.invokeAndWait(AccessBridge.java:7278)
	at jdk.accessibility/com.sun.java.accessibility.internal.AccessBridge$InvocationUtils.invokeAndWait(AccessBridge.java:7253)
	at jdk.accessibility/com.sun.java.accessibility.internal.AccessBridge.getAccessibleTextLineLeftBoundsFromContext(AccessBridge.java:2073)
	at jdk.accessibility/com.sun.java.accessibility.internal.AccessBridge.runDLL(Native Method)
	at jdk.accessibility/com.sun.java.accessibility.internal.AccessBridge$dllRunner.run(AccessBridge.java:177)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NullPointerException
	at androidx.compose.ui.platform.ComposeAccessible$ComposeAccessibleComponent$ComposeAccessibleText.getCharacterBounds(ComposeAccessible.kt:465)
	at jdk.accessibility/com.sun.java.accessibility.internal.AccessBridge$87.call(AccessBridge.java:2081)
	at jdk.accessibility/com.sun.java.accessibility.internal.AccessBridge$87.call(AccessBridge.java:2073)
	at jdk.accessibility/com.sun.java.accessibility.internal.AccessBridge$InvocationUtils$CallableWrapper.run(AccessBridge.java:7334)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:308)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

--- a/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/modifiers/TextStringSimpleNode.kt
+++ b/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/modifiers/TextStringSimpleNode.kt
@@ -199,7 +199,6 @@ internal class TextStringSimpleNode(
                     textLayoutResult.add(it)
                 }
                 layout != null
-                false
             }
             semanticsTextLayoutResult = localSemanticsTextLayoutResult
         }

--- a/compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/platform/AccessibilityTest.kt
+++ b/compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/platform/AccessibilityTest.kt
@@ -16,17 +16,11 @@
 
 package androidx.compose.ui.platform
 
-import androidx.compose.foundation.focusable
-import androidx.compose.foundation.layout.height
 import androidx.compose.material.Button
 import androidx.compose.material.LinearProgressIndicator
 import androidx.compose.material.Tab
 import androidx.compose.material.TabRow
 import androidx.compose.material.Text
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
-import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.assertThat
 import androidx.compose.ui.isEqualTo
@@ -36,7 +30,7 @@ import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.test.SemanticsNodeInteraction
 import androidx.compose.ui.test.junit4.createComposeRule
 import androidx.compose.ui.test.onNodeWithTag
-import androidx.compose.ui.unit.dp
+import java.awt.Point
 import javax.accessibility.AccessibleRole
 import javax.accessibility.AccessibleText
 import kotlin.test.fail
@@ -74,6 +68,9 @@ class AccessibilityTest {
         assertEquals("d", accessibleText.getBeforeIndex(AccessibleText.CHARACTER, 21))
         assertEquals("world", accessibleText.getBeforeIndex(AccessibleText.WORD, 21))
         assertEquals("Hi world", accessibleText.getBeforeIndex(AccessibleText.SENTENCE, 21))
+
+        assertEquals(0, accessibleText.getIndexAtPoint(Point(0, 0)))
+        assertEquals("Hello world. Hi world.".length, accessibleText.getIndexAtPoint(Point(10000, 10000)))
     }
 
     @Test

